plugins/topology/dev/index.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { EntityProvider } from '@backstage/plugin-catalog-react';
+import { Entity } from '@backstage/catalog-model';
+import { createDevApp } from '@backstage/dev-utils';
+import { topologyPlugin, TopologyPage } from '../src/plugin';
+import {
+  EntityKubernetesContent,
+  KubernetesApi,
+  kubernetesApiRef,
+} from '@backstage/plugin-kubernetes';
+import { TestApiProvider } from '@backstage/test-utils';
+import fixture1 from '../src/__fixtures__/1-deployments.json';
```

Comment:
```fixture1``` variable seems odd i suggest using a variable name which is little more understandable